doc/nrf/includes/thread_configure_network.txt

```diff
@@ -0,0 +1,10 @@
+Configure the required Thread network parameters with the ``ot channel``, ``ot panid`` and ``ot masterkey`` commands:
```

Comment:
```suggestion
Configure the required Thread network parameters with the ``ot channel``, ``ot panid``, and ``ot masterkey`` commands:
```

Reply:
Where do I get the channel, panid, and master key from?
I see in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thread_commissioning.html#setting-up-the-commissioner how I can check the parameters, but that is after I start the network.
Do I just specify random values? Or are they defined somewhere?

Reply:
I used default parameters present in Zephyr:

Panid:
https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/net/l2/openthread/Kconfig.thread#L8
(43981 is 0xabcd).

Masterkey:
https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/net/l2/openthread/Kconfig.thread#L31

Channel:
https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/net/l2/openthread/Kconfig.thread#L12

Reply:
Would it work with other values? Or do you have to use the ones that are configured in Kconfig (because those are the ones that the sample uses)?

Reply:
Yes, it would work. The point is to have these three set to the same values on all nodes that we want to connect to a single network.